INSTALL.txt

```diff
@@ -52,19 +52,25 @@ Specific guides to install from source are available for Windows and macOS:
     https://sabnzbd.org/wiki/installation/install-macos
     https://sabnzbd.org/wiki/installation/install-from-source-windows
 
-Only Python 3.5 and above is supported. Replace python3 with python in the commands
-below if Python 3 is the only version of Python on your system.
+Only Python 3.5 and above is supported.
 
-Install all required Python packages by running:
-    python3 -m pip install -r requirements.txt
+You need to install
+    par2 unrar unzip python3-setuptools python3-pip
```

Comment:
This is only for non-Windows/macOS systems:
`On Linux systems you need to install:`

Reply:
We're talking here about "4) INSTALL with only sources", not installing from the packages. So, on Windows and MacOS ... don't you need to install unrar etc?

Reply:
No, these are included here in the repo and when you install Python on Windows it already includes setuptools and pip.